apis/kueue/v1beta1/workload_conversion_test.go

```diff
@@ -260,6 +344,32 @@ func TestWorkloadConversion_RoundTrip(t *testing.T) {
 				},
 			},
 		},
+		"Workload with PodPriorityClassSource": {
+			v1beta1Obj: &Workload{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "test-workload",
+					Namespace: "default",
+				},
```

Comment:
nit, please use `defaultObjectMeta` for new cases, and the pre-existing ones too

Reply:
Done